### ISSUE TITLE
Docs: Code snippet is cut off on page 60 in the PDF

### ISSUE THREAD

**Issue by ACPK** [microsoft/TypeScript-Website#2131]
For PDF created on Monday, November 1, 2021 against commit 5983a1 with TypeScript 4.4.

The code snippet on the top of page 60 reads: 

    function map<Input, Output>(arr: Input[], func: (arg: Input) => Output): O
        return arr.map(func);
    }

Based on the documentation (https://www.typescriptlang.org/docs/handbook/2/functions.html#inference), it should be:

    function map<Input, Output>(arr: Input[], func: (arg: Input) => Output): Output[] {
      return arr.map(func);
    }

**Comment by orta** [comment]
`function map<Input, Output>(arr: Input[], func: (arg: Input) => Output): O` is 74 chars long, I think we should maybe look at running prettier with 72 chars so that it neatly fits with the horizontal constraints
